analysis.go

```diff
@@ -0,0 +1,53 @@
+package goInspector
```

Comment:
We need to put a standard header comment there

Reply:
@pombredanne can you give an example of such a comment?

Reply:
Check scancode-toolkit source code header comment